core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -1230,6 +1230,27 @@ class PlaintextConsumerTest extends BaseConsumerTest {
     assertEquals(500, consumer0.committed(tp2).offset)
   }
 
+  @Test
+  def testCommitInvalidOffsets() {
```

Comment:
This might be a little cheaper to test in `KafkaConsumerTest` if you can stand the mocking. If possible, it would be nice to try to avoid adding unnecessarily to the already painful build times.